ratatool-scalacheck/src/test/scala/com/spotify/ratatool/scalacheck/AvroGeneratorTest.scala

```diff
@@ -46,29 +40,25 @@ object AvroGeneratorTest extends Properties("AvroGenerator") {
       _.getRequiredFields.setLogicalDecimalField
     )
 
-  val richTupGen = (specificRecordOf[TestRecord], specificRecordOf[TestRecord]).tupled
-    .amend2(specificRecordOf[RequiredNestedRecord])(_.setRequiredFields, _.setRequiredFields)
+  val genRichTup = (genTestRecord, genTestRecord).tupled
+    .amend2(genRequiredNestedRecord)(_.setRequiredFields, _.setRequiredFields)
 
-  property("support RichAvroGen") = forAll(richGen) { r =>
+  property("support RichAvroGen") = forAll(genRich) { r =>
+    // format: off
```

Comment:
Any reason to exclude this from formatting?

Reply:
It looks nicer to have all test case as one liner even if this exceed max line with manually aligned custom `|:` operator